### PR TITLE
Adding CS Projections to Toast Notifications

### PR DESCRIPTION
**Pull request by loneursid**
Adds CS Projections to Toast Notifications

### CODE REVIEW

File: build/CopyFilesToStagingDir.ps1 [hunk 1]
```diff
@@ -85,6 +86,9 @@ PublishFile $FullBuildOutput\Microsoft.Windows.ApplicationModel.WindowsAppRuntim
 PublishFile $FullBuildOutput\Microsoft.Windows.ApplicationModel.WindowsAppRuntime.Projection\Microsoft.Windows.ApplicationModel.WindowsAppRuntime.Projection.pdb $NugetDir\lib\net5.0-windows10.0.17763.0
 PublishFile $FullBuildOutput\Microsoft.Windows.AppLifecycle.Projection\Microsoft.Windows.AppLifecycle.Projection.dll $NugetDir\lib\net5.0-windows10.0.17763.0
 PublishFile $FullBuildOutput\Microsoft.Windows.AppLifecycle.Projection\Microsoft.Windows.AppLifecycle.Projection.pdb $NugetDir\lib\net5.0-windows10.0.17763.0
+PublishFile $FullBuildOutput\Microsoft.Windows.AppNotifications.Projection\Microsoft.Windows.AppNotifications.Projection.dll $NugetDir\lib\net5.0-windows10.0.17763.0
+PublishFile $FullBuildOutput\Microsoft.Windows.AppNotifications.Projection\Microsoft.Windows.AppNotifications.Projection.pdb $NugetDir\lib\net5.0-windows10.0.17763.0
+PublishFile $FullBuildOutput\Microsoft.Windows.PushNotifications.Projection\Microsoft.Windows.PushNotifications.Projection.dll $NugetDir\lib\net5.0-windows10.0.17763.0
```
Comment (danielayala94) [inline build/CopyFilesToStagingDir.ps1:91]: Interesting, so we weren't publishing the projection dll for Push before?

**Review by danielayala94 — APPROVED** [review]
:shipit:

File: build/CopyFilesToStagingDir.ps1 (hunk 1 above)
Comment (loneursid) [reply build/CopyFilesToStagingDir.ps1:91]: nope, and no one ever noticed it...

**Review by hulumane — APPROVED** [review]

**Pull request merged**